rails/db/seeds.rb

```diff
@@ -45,8 +45,8 @@
                      point: flaming_forest,
                      permission_level: 1)
 
-User.where(email: 'admin@terrastories.com')
-    .first_or_create do |admin|
-      admin.password = 'password',
-      admin.role = 1
-    end
+User.where(email: 'admin@terrastories.com').first_or_create do |admin|
```

Comment:
Also to be considered is updating `first_or_create` to the bang-method version `first_or_create!`. This would have the `db:seed` rake task error with an exception if an `admin@terrastories.com` user cannot be found or _successfully created_ (i.e. this admin user seed falls out-of-sync with user model validations).